stable/osg-hosted-ce/osg-hosted-ce/README.md

```diff
@@ -267,6 +267,36 @@ To avoid these rate limits, it's possible to bootstrap the certificate request p
           HostKeySecret: <YOUR HOST KEY SECRET NAME>
           HostCertSecret: <YOUR HOST CERT SECRET NAME>
 
+### NetworkPolicy
+
+A generic NetworkPolicy for restricting traffic to and from the HostedCE. Allows configuration based on a list of IP CIDR ranges.
+
+By default NetworkPolicy is disabled and the application is open to all traffic.
+
+Network Policy can be thought of like a firewall for the application.
+
+Traffic can be restricted to only allow communication between the CE, OSG Factory, and your own cluster environment.
+
+```
+NetworkPolicy:
+  Enabled: false
+  AllowedCIDRs:
+  - 0.0.0.0/0
+```
+
+Multiple CIDR ranges can be allowed in a list like so:
+
+```
+NetworkPolicy:
+  Enabled: false
+  AllowedCIDRs:
+  - 0.0.0.0/0
+  - 10.0.0.0/8
+  - 192.168.0.0/16 
```

Comment:
If `0.0.0.0/0` is all ranges, wouldn't this allow unrestricted ingress? Also, we should probably set `Enabled: True` here.

Reply:
Yes. The default policy is wide open and disabled. I am amenable to `Enabled: True` being default if that's what we want. We can also remove `0.0.0.0/0` if we have a good default range.

Reply:
I don't think we should change enable this by default. Just that the documentation reads as an example of how to enable a network policy so the docs should 1. set `Enabled: True` and 2. remove `0.0.0.0/0` because presumably if they're enabling this, they want to restrict access